test/functional/test_framework/mempool_util.py

```diff
@@ -25,8 +28,8 @@ def fill_mempool(test_framework, node, miniwallet):
     It will not ensure mempools become synced as it
     is based on a single node and assumes -minrelaytxfee
     is 1 sat/vbyte.
-    To avoid unintentional tx dependencies, it is recommended to use separate miniwallets for
-    mempool filling vs transactions in tests.
+    To avoid unintentional tx dependencies, the mempool filling txs are created with a
+    separate tagged ephermal miniwallet instance.
```

Comment:
901ffa9e6e57f7c0e0db967fba9c715139bbcf0b nit: did you mean ephemeral? (also in commit message)